about.html

```diff
@@ -0,0 +1,99 @@
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="projects.html">Mes projets</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="contact.html">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <main class="container my-3">
+        <div class="container my-3 text-center">
+            <h1>À propos</h1>
+        </div>
+
+        <div class="container">
+            <div class="row">
+                <div class="col">
+                    <img class="img-fluid rounded mx-auto mb-2 d-block h-75" src="assets/portrait.jpg" alt="">
+                </div>
+                <div class="col">
+                    <p class="text-justify">Bonjour je suis un étudiant en développment web qui est prévue d'être diplômé en mai 2024. J'ai un AEC en programmation d'analyse</p>
+                </div>
+            </div>
+        </div>
+
+        <div class="container">
+            <h2 class="text-center">CV</h2>
+            <div class="row mb-3">
+                <div class="col border pt-2">
+                    <h3 class="text-center">Études</h3>
+                    <p class="text-justify">AEC - Programmeur analyste en informatique de gestion - 2022</p>
+                </div>
+            </div>
+            <div class="row mb-3">
+                <div class="col border pt-2">
+                    <h3 class="text-center mt-2">Professionnel</h3>
+                    <p class="text-justify">Technicien informatique - MAPAQ - Juillet 2020 à Janvier 2021</p>
+                    <p class="text-justify">Stagiaire en informatique - MAPAQ - Février 2020 à Mars 2020</p>
+                </div>
+            </div>
+        </div>
+    </main>
+
+    <footer class="container-fluid w-100 px-4 text-center border position-sticky-bottom">
+        <div class="col">
+            <div class="row">
+                <h5 class="mt-3">Liens</h5>
+                <ul class="list-group list-group-horizontal d-flex justify-content-center">
+                    <li class="list-group-item border-0"><a href="index.html">Home</a></li>
+                    <li class="list-group-item border-0"><a href="about.html">À propos</a></li>
+                    <li class="list-group-item border-0"><a href="projects.html">Mes projets</a></li>
+                    <li class="list-group-item border-0"><a href="contact.html">Contact</a></li>
+                </ul>
+            </div>
+            <div class="row">
+                <h5>Mes réseaux sociaux</h5>
+                <ul class="list-group list-group-horizontal d-flex justify-content-center">
+                    <li class="list-group-item border-0"><a href="https://www.facebook.com/">Facebook</a></li>
```

Comment:
For external URL's, use `target='_blank'` property. This will make that your page will open in a new tab, instead in the same page.